code/FulpstationCode/fulp_emag/fulp_emag.dm

```diff
@@ -55,7 +58,23 @@
 
 	if(istype(A, /mob/living/silicon/robot))
 		var/mob/living/silicon/robot/R = A
-		if(!R.emagged)
-			return TRUE
+		if(pre_check && (R.opened || R.locked) ) //Checks specific to pre emag act.
+			return FALSE //If the cover is opened, we will always pass false for the pre-check and true for the post-check as borgs can be subverted any number of times.

```

Comment:
isn't there technically, like, a 10 second cooldown per borg for subverting a borg with an emag, so they don't get spammed with messages?

eh, it doesn't really matter, and trying to "patch" this one edge case would cause more problems than it'd solve